homeassistant/components/bosch_shc/config_flow.py

```diff
@@ -0,0 +1,220 @@
+"""Config flow for Bosch Smart Home Controller integration."""
+import logging
+from os import makedirs
+
+from boschshcpy import SHCRegisterClient, SHCSession
+from boschshcpy.exceptions import (
+    SHCAuthenticationError,
+    SHCConnectionError,
+    SHCmDNSError,
+    SHCRegistrationError,
+    SHCSessionError,
+)
+import voluptuous as vol
+
+from homeassistant import config_entries, core
+from homeassistant.components.zeroconf import async_get_instance
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_TOKEN
+
+from .const import (
+    CONF_HOSTNAME,
+    CONF_SHC_CERT,
+    CONF_SHC_KEY,
+    CONF_SSL_CERTIFICATE,
+    CONF_SSL_KEY,
+)
+from .const import DOMAIN  # pylint:disable=unused-import
```

Comment:
We don't need this pylint disable anymore. It's been fixed upstream.
```suggestion
from .const import (
    CONF_HOSTNAME,
    CONF_SHC_CERT,
    CONF_SHC_KEY,
    CONF_SSL_CERTIFICATE,
    CONF_SSL_KEY,
    DOMAIN,
)
```